src/tea_html.res

```diff
@@ -400,7 +395,12 @@ let onWithOptions = (~key: string, eventName, options: options, decoder) =>
     if options.preventDefault {
       preventDefault(event) |> ignore
     }
-    event |> Tea_json.Decoder.decodeEvent(decoder) |> result_to_option
+    event |> Tea_json.Decoder.decodeEvent(decoder) |> (x =>
```

Comment:
Can you use something like:

```
let result = event |> Tea_json.Decoder.decodeEvent(decoder)
switch result {
  | Ok(a) => Some(a)
  | Error(_) => None
}
```

Reply:
I tried this and it didn't work, result wouldn't be recognizable in the switch statement
trying this also didn't work
    let result = event |> Tea_json.Decoder.decodeEvent(decoder)
      let result =switch result {
      | Ok(a) => Some(a)
      | Error(_) => None
      }